pkg/parser/quoted_string_test.go

```diff
@@ -0,0 +1,1009 @@
+package parser_test
+
+import (
+	"github.com/bytesparadise/libasciidoc/pkg/types"
+	. "github.com/bytesparadise/libasciidoc/testsupport"
+
+	. "github.com/onsi/ginkgo" //nolint golint
+	. "github.com/onsi/gomega" //nolint golint
+)
+
+var _ = Describe("quoted strings", func() {
+
+	Context("draft document", func() {
+		It("simple single quoted string", func() {
+			source := "'`curly was single`'"
+			expected := types.DraftDocument{
+				Blocks: []interface{}{
+					types.Paragraph{
+						Lines: [][]interface{}{
+							{
+								types.QuotedString{
+									Kind: types.SingleQuote,
+									Elements: []interface{}{
+										types.StringElement{Content: "curly was single"},
+									},
+								},
+							},
+						},
+					},
+				},
+			}
+			Expect(ParseDraftDocument(source)).To(MatchDraftDocument(expected))
+		})
+		It("spaces with single quoted string", func() {
+			source := "'` curly was single `'"
```

Comment:
do we allow spaces between the quotes and the text ?

Reply:
Yes.  This matches asciidoctor.

Reply:
ok, good to know ;)